src/components/Footer.js

```diff
@@ -1,19 +1,24 @@
 import React from 'react'
 
 const Footer = () => (
-  <div className='footer'>
-    <hr />
+  <div>
     <div className='formulas-info'>
-      Sposób obliczania punktów znajdziesz na stronie:&nbsp;&nbsp;
-      <a target='_blank'
-        href='http://www.agh.edu.pl/rekrutacja/studia-pierwszego-stopnia/rekrutacja-20152016/nowa-matura/'
-        alt='Link do strony AGH ze sposobem obliczania punktów'>
-        http://www.agh.edu.pl/rekrutacja/studia-pierwszego-stopnia/rekrutacja-20152016/nowa-matura/
-      </a>
+      <div className='formulas-info-content'>
+        Sposób obliczania punktów znajdziesz na stronie:&nbsp;&nbsp;
+        <a target='_blank'
+          href='http://www.agh.edu.pl/rekrutacja/studia-pierwszego-stopnia/rekrutacja-20152016/nowa-matura/'
+          alt='Link do strony AGH ze sposobem obliczania punktów'>
+          http://www.agh.edu.pl/rekrutacja/studia-pierwszego-stopnia/rekrutacja-20152016/nowa-matura/
```

Comment:
haha, no hate mate.
I just haven't noticed the closing of `a` tag, and thought this `http:` was it's property. Even though `a` don't have `http` property. That's why I was confused.